### ISSUE TITLE
Serial number on listing modal

### ISSUE THREAD

**Issue by prury** [kodadot/nft-gallery#9971]
after #9956 gets merged the following needs to be fixed: 

> sn on listing:

![318809066-6d74ec75-5f2c-4298-89e1-1bee3e1c386e](https://github.com/kodadot/nft-gallery/assets/36627808/0ce9c5b7-fa69-412d-b1cd-ffc6becb0744)

let's create followup issue for this. If metadata is still updating, we need to override the `list-store` with `drop-store`

<img src="https://github.com/kodadot/nft-gallery/assets/734428/6018f0c5-b531-4a6e-a94c-7ba726e6cd74" width="300" />

_Originally posted by @preschian in https://github.com/kodadot/nft-gallery/issues/9956#issuecomment-2033515238_
